hw1/1.1/1.1/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace _1._1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("enter the size of the array:");
+            int length = Convert.ToInt32(Console.ReadLine());
+            int[] array = new int[length];
+            Console.WriteLine("enter the elements of the array:");
+            for (int i = 0; i < length; i++)
```

Comment:
```suggestion
            for (var i = 0; i < length; i++)
```
В циклах `for` по целочисленной переменной так принято